bandit/bandit_report.js

```diff
@@ -12,12 +12,15 @@ function createMoreInfoLinks (issues) {
   let moreInfo = 'For more information about the issues follow the links: \n'
 
   for (let issue of issues) {
-    if (issuesMap.has(issue.test_id) === false) {
+    if (issuesMap.has(issue.test_id) === false && issue.more_info) {
       issuesMap.set(issue.test_id)
       const text = `${issue.test_id}: ${issue.test_name}`
       moreInfo += `[${text}](${issue.more_info})\n`
     }
   }
+  if (issuesMap.size <= 0) {
```

Comment:
curious, does javascript ever let a size be less than 0?

Reply:
I read a little more about this and all answers says that it can't be negative BUT there is one curious case:
![image](https://user-images.githubusercontent.com/16246778/53866222-b35af500-3ff9-11e9-9237-9b533a46a3b1.png)

mentioned here: https://stackoverflow.com/questions/32386253/can-the-js-array-length-property-ever-return-a-negative-value